ui/src/data/source.tsx

```diff
@@ -161,30 +155,27 @@ export type FeatureSet = {
   output: FeatureSetOutput[];
 };
 
+export type EntityGroupData = {
+  id: string;
+  entityId: string;
+  relatedEntityId?: string;
+  relatedEntityGroupId?: string;
+  occurrenceEntityIds: string[];
```

Comment:
Are these entity/group ids the same as the names in the config file?

Reply:
Yes